quint/io-cli-tests.md

```diff
@@ -278,31 +275,28 @@ recorded yet.
 <!-- !test exit 1 -->
 <!-- !test in test empty trace -->
 ```
-output=$(quint test --seed 1 --verbosity=3 ../examples/language-features/counters.qnt)
+output=$(quint test --seed 1 --verbosity=3 ./testFixture/simulator/failingTestCounters.qnt)
```

Comment:
Just curious, why did you move this test to the fixtures? Does not it demonstrate `assert`?

Reply:
It’s explained in the commit message for posterity: we have an example of assert in the example version. But if we include failing tests in the examples then they will fail when we run the tests. So the failing test in the example was changed into a test with `fails`, and the test of test failure was moved into `testFixture`. :)